dashboard/app/assets/javascripts/admin.js.erb

```diff
@@ -0,0 +1,26 @@
+/**
+ * Dynamic generation and event bindings for project admin section of the admin box
+ */
+
+if (!window.dashboard) {
+  window.dashboard = {};
+}
```

Comment:
oh, cool, I copied this from header.js, is it necessary there for some reason or is that vestigial?